.github/workflows/k8s-validation.yaml

```
@@ -1,6 +1,12 @@
 name: K8s validation
 on:
   workflow_call:
+    inputs:
+      environments:
+        description: "Comma-separated list of environments to validate (e.g., staging,production)"
+        default: "staging,production"
```

Und hier dann entsprechend so (siehe unten 😊):
```suggestion
        description: "space-separated list of environments to validate (e.g. 'staging production')"
        default: "staging production"
```